accounts/forms.py

```diff
@@ -39,7 +39,7 @@ class RegisterAccountForm(UserCreationForm):
     subscribe_news = forms.BooleanField(
         required=False,
         initial=True,
-        label="Get exclusive updates before they're public",
+        label="Get exclusive updates on artists",
```

Comment:
I think this label refers to the newsletter, but by providing this label it may confuse users into think these are the artist updates.

Reply:
Yeah, I agree it's a little vague. Maybe adding "and new projects". Was trying to be more specific on what kind of updates they would get. I think they would mostly be when new artists